app/Http/Controllers/LiquidationController.php

```
@@ -0,0 +1,81 @@
+                'stakeholder_id' => $stakeholder->id ?? null
+            ];
+        }
+        $validatedData['details'] = $newDetails;
+        $paymentRequest->details()->createMany($validatedData['details']);
+        return response()->json([
+            'message' => 'Liquidation created successfully',
+            'data' => $paymentRequest
+        ], 201);
+    }
+
+    public function update(Request $request, $id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)